spyderlib/plugins/configdialog.py

```diff
@@ -704,6 +752,13 @@ def setup_page(self):
         margins_layout.addWidget(margin_spin)
         prompt_box = newcb(_("Prompt when exiting"), 'prompt_on_exit')
 
+        # Language chooser
+        langs = get_available_translations()
+        choices = list(zip([LANGUAGE_CODES[key] for key in langs], langs))

```

Comment:
I copied this from code above (the choices part and adapted it...) 

It holds what is stored in `LANGUAGE_CODES` in the `baseconfig.py` 

So the combo box display Español, English... etc... and the value is ...es, en... etc..